demo/main_activity.hpp

```diff
@@ -23,4 +23,6 @@ class MainActivity : public brls::Activity
   public:
     // Declare that the content of this activity is the given XML file
     CONTENT_FROM_XML_RES("activity/main.xml");
+
+    void onContentAvailable() override;
```

Comment:
The demo aims to be used as a reference and learn by example, can you add a comment to explain what it does / why it's here? Like I did for the CONTENT_FROM_XML_RES above

Reply:
Done. :+1: